aws/resource_aws_spot_instance_request.go

```diff
@@ -132,6 +144,22 @@ func resourceAwsSpotInstanceRequestCreate(d *schema.ResourceData, meta interface
 		spotOpts.LaunchGroup = aws.String(v.(string))
 	}
 
+	if v, ok := d.GetOk("valid_from"); ok {
+		valid_from, err := time.Parse(awsSpotInstanceRequestTimeLayout, v.(string))
+		if err != nil {
+			return err
+		}
+		spotOpts.ValidFrom = &valid_from
```

Comment:
Minor nitpick: we prefer to use the AWS SDK provided functions for these, e.g. `aws.Time(valid_from)`

Reply:
Used the aws.Time function for pointer reference.